package.json

```diff
@@ -8,7 +8,7 @@
     "url": "git+https://github.com/anthonygore/html-critical-webpack-plugin.git"
   },
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">=10.0.0"
```

Comment:
is this change a requirement because of critical?  Does it have a new minimum version now?  Not that I am opposed, but I think this would definitely necessitate a semver major at minimum  from this package then, when making the next release.

Reply:
Yes. Critical has node >=10 set as engine. See https://github.com/addyosmani/critical/blob/master/package.json#L26

Reply:
Cool, that's what I figured.  Thanks for confirming.  👍